### ISSUE TITLE
Plugin did not work on Mohist

### ISSUE THREAD

**Issue by KLONOA9X6** [Jannyboy11/InvSee-plus-plus#83]
InvSee++ Version: 0.24.0
Mohist version: 1.20.1-426
logs:
```
......
[01:14:11] [Server thread/INFO] [com.janboerman.invsee.spigot.InvseePlusPlus]: [InvSee++] Enabling InvSeePlusPlus v0.24.0
[01:14:11] [Server thread/ERROR] [Minecraft]: Error occurred while enabling InvSeePlusPlus v0.24.0 (Is it up to date?)
java.lang.ClassFormatError: Duplicate method name "m_150399_" with signature "(IILnet.minecraft.world.inventory.ClickType;Lnet.minecraft.world.entity.player.Player;)V" in class file com/janboerman/invsee/spigot/impl_1_20_1_R1/MainNmsContainer
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:1017) ~[?:?]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:240) ~[forge-1.20.1-47.1.76-universal.jar%23308!/:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:592) ~[?:?]
	at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:143) ~[forge-1.20.1-47.1.76-universal.jar%23308!/:?]
	at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:138) ~[forge-1.20.1-47.1.76-universal.jar%23308!/:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?]
	at com.janboerman.invsee.spigot.Impl_1_20_1.<init>(Setup.java:112) ~[?:?]
	at com.janboerman.invsee.spigot.Setup.setup(Setup.java:50) ~[?:?]
	at com.janboerman.invsee.spigot.InvseePlusPlus.onEnable(InvseePlusPlus.java:79) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:277) ~[forge-1.20.1-47.1.76-universal.jar%23308!/:?]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:336) ~[forge-1.20.1-47.1.76-universal.jar%23308!/:?]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:482) ~[forge-1.20.1-47.1.76-universal.jar%23308!/:?]
	at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugin(CraftServer.java:547) ~[forge-1.20.1-47.1.76-universal.jar%23308!/:997de31d-ddc9a2dad-d2eba2c8-47.1.47]
	at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugins(CraftServer.java:460) ~[forge-1.20.1-47.1.76-universal.jar%23308!/:997de31d-ddc9a2dad-d2eba2c8-47.1.47]
	at net.minecraft.server.MinecraftServer.m_129815_(MinecraftServer.java:437) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?]
	at net.minecraft.server.MinecraftServer.m_130006_(MinecraftServer.java:339) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?]
	at net.minecraft.server.dedicated.DedicatedServer.m_7038_(DedicatedServer.java:219) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?]
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:851) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?]
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:279) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
[01:14:11] [Server thread/INFO] [com.janboerman.invsee.spigot.InvseePlusPlus]: [InvSee++] Disabling InvSeePlusPlus v0.24.0
......
```

**Comment by Jannyboy11** [comment]
What makes you think this is a bug in invSee++? To me this looks like Mohist's plugin class bytecode transformer has a bug.

**Comment by Jannyboy11** [comment]
Despite this not being a bug in InvSee++, this issue should be resolved on 1.20.2: https://github.com/Jannyboy11/InvSee-plus-plus/releases/tag/v0.24.1
